### PR TITLE
Revising README

### PR DESCRIPTION
**Pull request by rajkaramchedu**
Are there other ways to install, like from PyPI package? If so, @galina-opentensor, fyi.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,143 @@
...
+  EMISSION: true
+  HOTKEY: true
+  INCENTIVE: true
+  RANK: true
+  STAKE: true
+  TRUST: true
+  UID: true
+  UPDATED: true
+  VAL: true
+  VTRUST: true
```
Comment (thewhaleking) [inline README.md:124]: This should generally not be viewed/edited directly by the user.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,143 @@
...
+---
+
+## Install on Windows
+
+To install and run Bittensor SDK on Windows you must install [**WSL 2** (Windows Subsystem for Linux)](https://learn.microsoft.com/en-us/windows/wsl/about) on Windows and select [Ubuntu Linux distribution](https://github.com/ubuntu/WSL/blob/main/docs/guides/install-ubuntu-wsl2.md). 
+
+After you installed the above, follow the same installation steps described above in [Install on macOS and Linux](#install-on-macos-and-linux).
+
+**ALERT**: **Limited support on Windows for Bittensor operations**
+While wallet transactions like delegating, transfer, registering, staking can be performed on a Windows machine using WSL 2, the subnet mining and subnet validating operations are not recommended and are not supported on Windows machines.
```
Comment (thewhaleking) [inline README.md:80]: Has this been tested at all?

**Comment by thewhaleking** [comment]
This will be available on PyPI (see: https://github.com/opentensor/btcli/pull/114). It is not at this very moment, but we should have a planned part of this doc ready to go when the release actually happens.

File: README.md (hunk 2 above)
Comment (galina-opentensor) [reply README.md:80]: Yes, I have tested it. Did I miss anything?

**Review by thewhaleking — APPROVED** [review]

**Review by galina-opentensor — APPROVED** [review]

**Pull request merged**